include/gazebo_motor_failure_model.h

```diff
@@ -0,0 +1,179 @@
+/*
```

Comment:
This is a copy-paste of the `gazebo_motor_model` plugin. This is not what is pretended. You have to create a plugin that does not replace `gazebo_motor_model` but either interacts with it.